test/system/irohad_test_data/config.sample.copy

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
What did you add this file for?

Reply:
I didn't. This is pretty annoying, some test modifies this file. I will fix it one day.

Reply:
Well, it wasn't in `.gitignore` for some reason so easy fix is coming.